CollAction/Models/ApplicationUser.cs

```diff
@@ -28,18 +28,14 @@ public class ApplicationUser : IdentityUser
 
         public async Task SetNewsletterSubscription(ApplicationDbContext context, string email, bool wantsSubscription)
         {
-            NewsletterSubscription subscription = await context.NewsletterSubscriptions.SingleOrDefaultAsync(s => s.Email.Equals(email, StringComparison.Ordinal));
+            NewsletterSubscription subscription = await context.NewsletterSubscriptions.SingleOrDefaultAsync(s => s.Email.ToLower() == email.ToLower());
```

Comment:
This won't use the index you created on email. We should change the type to citext to ensure you don't need to lower, or accept that we can't do this efficiently for now. See: https://www.postgresql.org/docs/9.1/static/citext.html

Sorry, should've realized this earlier.